scripts/build-aci

```diff
@@ -1,11 +1,42 @@
-#!/usr/bin/bash
+#!/bin/bash
 
-if [ ! -d rootfs ]; then
-	mkdir -p rootfs
-	mv * rootfs
+# A non-installed actool can be used, for example:
+# ACTOOL=../../appc/spec/bin/actool
+ACTOOL=${ACTOOL:-actool}
+
+IMAGEDIR=${IMAGEDIR:-bin/image-aci}
+
+VERSION=$1
+
+if [ ! -x "$ACTOOL" ] ; then
+    echo "actool ($ACTOOL) is not executable"
+    exit 1
+fi
+
+if [ ! -x bin/etcd ] ; then
+    echo "bin/etcd not found. Is it compiled?"
+    exit 1
+fi
+
+if [ -z "$VERSION" ] ; then
+    echo "Usage: scripts/build-aci VERSION"
+    exit 1
+fi
+
+mkdir -p $IMAGEDIR/rootfs
+if [ ! -d $IMAGEDIR/rootfs -o ! -x $IMAGEDIR/rootfs ]; then
+    echo "$IMAGEDIR/rootfs is not a writeable directory"
+    exit 1
 fi
 
-cat <<DF > manifest
+if [ -n "$(ls -A $IMAGEDIR/rootfs)" ]; then
+    echo "$IMAGEDIR/rootfs is not empty"
+    exit 1
+fi
+
+cp bin/* $IMAGEDIR/rootfs
```

Comment:
Rather explicitly list the files. For reference, this is the current etcd aci:

```
drwxr-xr-x 501/20            0 2014-12-18 10:51 rootfs
drwxr-xr-x 501/20            0 2014-12-18 10:32 rootfs/Documentation
-rw-r--r-- 501/20         2266 2014-12-18 10:32 rootfs/Documentation/0_4_migration_tool.md
-rw-r--r-- 501/20         6770 2014-12-18 10:32 rootfs/Documentation/admin_guide.md
-rw-r--r-- 501/20        25262 2014-12-18 10:32 rootfs/Documentation/api.md
-rw-r--r-- 501/20         3105 2014-12-18 10:32 rootfs/Documentation/backward_compatibility.md
-rw-r--r-- 501/20        11683 2014-12-18 10:32 rootfs/Documentation/clustering.md
-rw-r--r-- 501/20         4598 2014-12-18 10:32 rootfs/Documentation/configuration.md
-rw-r--r-- 501/20          753 2014-12-18 10:32 rootfs/Documentation/glossary.md
-rw-r--r-- 501/20         3303 2014-12-18 10:32 rootfs/Documentation/other_apis.md
-rw-r--r-- 501/20         2751 2014-12-18 10:32 rootfs/Documentation/proxy.md
-rw-r--r-- 501/20         7053 2014-12-18 10:32 rootfs/Documentation/runtime-configuration.md
-rw-r--r-- 501/20         5186 2014-12-18 10:32 rootfs/README-etcdctl.md
-rw-r--r-- 501/20         4788 2014-12-18 10:32 rootfs/README.md
drwxr-xr-x 501/20            0 2014-12-18 10:51 rootfs/etc
-rw-r--r-- 501/20           79 2014-12-18 10:51 rootfs/etc/hosts
-rwxr-xr-x 501/20      9159736 2014-12-18 10:32 rootfs/etcd
-rwxr-xr-x 501/20      7199704 2014-12-18 10:32 rootfs/etcd-migrate
-rwxr-xr-x 501/20      6266752 2014-12-18 10:32 rootfs/etcdctl
-rw-r--r-- root/root       374 2014-12-18 10:51 manifest
```

I think the documentation/readme could be useful to keep in there for now. We definitely don't need `etcd-migrate`, though.